internal/config/theme.go

```diff
@@ -16,16 +16,16 @@ type (
 	}
 
 	GuildsTreeTheme struct {
-		AutoExpandFolders bool   `toml:"auto_expand_folders"`
-		Graphics          bool   `toml:"graphics"`
-		DmColor           string `toml:"dm_color"`
-		FolderColor       string `toml:"folder_color"`
-		ChannelColor      string `toml:"channel_color"`
+		AutoExpandFolders   bool   `toml:"auto_expand_folders"`
+		ChannelColor        string `toml:"channel_color"`
+		Graphics            bool   `toml:"graphics"`
```

Comment:
Why is the `Graphics` field shifted down?

Reply:
sorted alphabetically

Reply:
Should I change it back?